Tangem/Modules/Send/Summary/SendSummaryViewModel.swift

```diff
@@ -19,6 +19,7 @@ protocol SendSummaryViewModelInput: AnyObject {
     var destinationTextPublisher: AnyPublisher<String, Never> { get }
     var additionalFieldPublisher: AnyPublisher<(SendAdditionalFields, String)?, Never> { get }
     var feeValuePublisher: AnyPublisher<Fee?, Never> { get }
+    var feeValues: AnyPublisher<[FeeOption: LoadingValue<Fee>], Never> { get }
```

Comment:
а так может лучше? 
вроде же все опции одновременно приходят
```suggestion
    var feeValues: AnyPublisher<LoadingValue<[FeeOption : Fee]>, Never> { get }
```